Immense.RemoteControl.Desktop.Shared/Services/Viewer.cs

```diff
@@ -309,14 +306,22 @@ private async Task TrySendToViewer<T>(T dto, DtoType type, string viewerConnecti
     {
         try
         {
+            if (!_desktopHubConnection.IsConnected)
+            {
+                _logger.LogWarning(
+                    "Unable to send DTO type {type} because the app is disconnected from the server.", 
+                    type);
+                return;
+            }
+
             foreach (var chunk in DtoChunker.ChunkDto(dto, type))
             {
                 await _desktopHubConnection.SendDtoToViewer(chunk, viewerConnectionId);
             }
         }
         catch (Exception ex)
         {
-            _logger.LogError(ex, "Error while sending to viewer.");
+            _logger.LogError(ex, "Error while sending DTO type {type} to viewer.", type);
```

Comment:
It might be useful to include the viewerConnectionId in the log output

Reply:
I'll add it in the next PR.  :)